playground/backend/internal/environment/environment_service_test.go

```diff
@@ -113,13 +120,13 @@ func Test_getSdkEnvsFromOsEnvs(t *testing.T) {
 		},
 		{
 			name:      "default beam envs",
-			want:      NewBeamEnvs(defaultSdk, NewExecutorConfig("javac", "java", []string{"-d", "bin", "-classpath", defaultBeamSdkPath}, []string{"-cp", "bin:" + jars}), preparedModDir),
+			want:      NewBeamEnvs(defaultSdk, executorConfig, preparedModDir),
 			envsToSet: map[string]string{beamSdkKey: "SDK_JAVA"},
 			wantErr:   false,
 		},
 		{
 			name:      "specific sdk key in os envs",
```

Comment:
Please add a comment for this test case.

Reply:
the same